frontend/src/components/commons/PlaceForm.js

```diff
@@ -0,0 +1,363 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Tooltip from '@material-ui/core/Tooltip';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import CancelOutlinedIcon from '@material-ui/icons/CancelOutlined';
+import SaveOutlinedIcon from '@material-ui/icons/SaveOutlined';
+import MenuItem from '@material-ui/core/MenuItem';
+
+const useStyles = makeStyles((theme) => ({
+  form: {
+    width: '100%', // Fix IE 11 issue.
+    marginTop: theme.spacing(3),
+  },
+  button: {
+    margin: theme.spacing(3, 0, 2),
+  },
+}));
+
+const initialState = {
+  primaryPictureUrl: '',
+  type: 'cityscape',
+  title: '',
+  street: '',
+  address: '',
+  latitude: '',
+  longitude: '',
+  placeDescription: '',
+  pictureDescription: '',
+  aperture: '',
+  focalLength: '',
+  shutterSpeed: '',
+  iso: '',
+  flash: '',
+  youTubeUrl: '',
+  extraOne: '',
+  extraTwo: '',
+  particularities: '',
+};
+
+const types = [
```

Comment:
These are the same types as in `FormInput/Types.js`, maybe move it into its own file and use it in both places?

Reply:
thanks, will look into that